island-is/island.is
# feat(samradsgatt): Search of case in Minar umsagnir

# Add search to Minar umsagnir page

https://veflausnir.atlassian.net/browse/KAM-769

## What

User can search his on reviewed cases based on name, type and institution.
This branch is created on top of feature/KAM-596-page-mvf-minar-umsagnir (please review that pull request first)

## Why

.....

## Screenshots / Gifs

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/106164521/218100849-b15316f1-5791-4076-937b-733e963e976e.png">

https://www.figma.com/file/gFSUsFCIMYWRGa94XsmhpT/DEV-Samr%C3%A1%C3%B0sg%C3%A1tt?node-id=101%3A4718&t=d2r3iqdyz0mP8SEU-0

## How to test

Checkout branch feature/KAM-769-feature-leit-a-umsognum-efitr-mali-etc
Run the project locally: yarn start samradsgatt
Navigate to http://localhost:4200/samradsgatt/umsagnir
Compare and observe

## Checklist:
